jina/peapods/pods/k8s.py

```diff
@@ -23,6 +23,8 @@ def __init__(
             tail_port_out: int,
             head_zmq_identity: bytes,
             version: str,
+            pea_type: str,
```

Comment:
why are we doing changes at this level?

Reply:
We need to know if a kubernetes deployment is a head or a 'worker' pea so we can map it correctly and send to the appropriate one when asked. I think there is no other way to find out for the k8s connection pool otherwise. Same goes for the jina_pod_name and the shard_id